src/FilterOptions.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import _ from 'lodash/fp'
+import F from 'futil-js'
+import { newNodeFromType } from './utils/search'
+
+let FilterOptions = ({ node, tree, fields, Item }) => (
```

Comment:
agreed with @stellarhoof - this doesn't mean much outside of `FilterList`, and `FilterActions` is probably a better name